src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -267,6 +267,9 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
                     return Promise.reject(new Error(error));                    
                 }
                 break;
+            case TrialConfigMetadataKey.DISABLE_LOG:
```

Comment:
Another question is, why should we put this config key in trial config section, rather than experiment config section, like multiphase. Since disable remote logging is for experiment, as if we put it into experiment config, we can avoid process this key in Remote/PAI/Frameworkluacher/k8S, etc.

Reply:
disableLog is removed.